src/energyplus/Test/ReverseTranslator_GTest.cpp

```diff
@@ -548,6 +551,34 @@ TEST_F(EnergyPlusFixture, ReverseTranslator_FoundationKivaSettings) {
   EXPECT_EQ("Hourly", foundationKivaSettings.simulationTimestep());
 }
 
+TEST_F(EnergyPlusFixture, ReverseTranslator_OutputTableSummaryReports) {
+  openstudio::Workspace workspace(openstudio::StrictnessLevel::None, openstudio::IddFileType::EnergyPlus);
+
+  openstudio::IdfObject idfObject(openstudio::IddObjectType::Output_Table_SummaryReports);
+  IdfExtensibleGroup group1 = idfObject.pushExtensibleGroup();
+  group1.setString(0, "LightingSummary");
+  IdfExtensibleGroup group2 = idfObject.pushExtensibleGroup();
+  group2.setString(0, "EquipmentSummary");
+  IdfExtensibleGroup group3 = idfObject.pushExtensibleGroup();
+  group3.setString(0, "HVACSizingSummary");
```

Comment:
Perhaps worth adding a duplicate report, to ensure the model still has unique values. (Similar to what I've done at https://github.com/NREL/OpenStudio/blob/92c9f11baf755c907e5b4c07df367f4dc9dcbbef/src/energyplus/Test/OutputDiagnostics_GTest.cpp#L92-L129)